app/services/update_site_setting.rb

```diff
@@ -9,38 +9,36 @@ class UpdateSiteSetting
     attribute :new_value
     attribute :allow_changing_hidden, :boolean, default: false
 
+    before_validation { self.setting_name = setting_name&.to_sym }
```

Comment:
I improved this service a bit, as I had to touch it.

Reply:
Nice one!